docs/guides/serverless.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Introduction to Qiskit Serverless
+description: Use Qiskit Serverless to run quantum-classical workloads across the cloud and on quantum-centric supercomputers.
+---
+
+# What is Qiskit Serverless?
+
+Qiskit Serverless provides a simple interface to run workloads across quantum-classical resources. This includes deploying programs to the IBM Quantum&trade; Platform, running workloads remotely, and easy resource management for multi-cloud and quantum-centric supercomputing use cases.
+
+Premium users can build, deploy, and run their workloads remotely on classical compute made available through IBM Quantum Platform. Try out the tutorials in [IBM Quantum Learning](https://learning.quantum.ibm.com/catalog/tutorials?topics=qiskit-patterns) (note: these are accessible in the Premium Plan once you have logged into your IBM Quantum account).
+
+<Admonition type="note">
+    This preview feature is under frequent development, subject to change.
+</Admonition>
+
+## When should I use Qiskit Serverless?
+
+Qiskit Serverless helps manage classical and quantum resources across the entire Qiskit pattern workflow. This includes some of the following examples:
+
+- Parallelizing classical tasks, such as pre-processing and post-processing
+- Persisting long-running workloads in the cloud, even if your laptop is turned off
+- Deploying reusable programs
+
+## Install Qiskit Serverless
+
+To use Qiskit Serverless on IBM Quantum Platform, install the `qiskit-serverless` package:
+
+```bash
+pip install qiskit-serverless
+```
+
+This package requires `python` 3.11+. For users and organizations who want to run Serverless on custom infrastructure, follow the [Cloud infrastructure setup](https://qiskit.github.io/qiskit-serverless/cloud.html) guide.
+
+## Next steps
+
+<Admonition type="info" title="Recommendations">
+
+Explore how to [write your first Qiskit Serverless program](./serverless-first-program).
```

Comment:
Are there any learning platform tutorials that use serverless currently? might be nice to link them here if they exist

Reply:
Yep but they need to be updated over the next few weeks